App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { InputUser } from "./InputUser";
+import { View } from "react-native";
+import { Headers } from "./Header";
+
+const App = () => (
+  <View>
+    <Headers />
+    <InputUser />
+  </View>
```

Comment:
Nice work here! But, what do you think of moving this screen to the appropriate file?

Let's structure de project a little bit:
1. Create a folder named `src` in the root directory
2. Inside the `src` folder, create a new one named `screens` (or `scenes`) and another named `components`
3. Move the components to its folder (e.g. `Headers` and `InputUser`), and change the import statement accordingly (e.g. `import { Headers } from 'src/components/Headers'`. The project should be configured with absolute paths if doing this way.)
4. Create a new screen component. Try to be creative and consistent with the design. Maybe `NewTask` is a good name for this one. Remember to create it under `src/screens`, and move the content from `App.jsx` (we are abstracting a component from some other components, or in this case, a screen).
5. Import the screen in the root file: `App`. Then, call it inside: `<NewTask />`.
6. Check that all the imports in every file are consistent and don't fail.